themes/ixTheoTheme/templates/record/pdasubscribe.phtml

```diff
@@ -0,0 +1,20 @@
+<?= $this->flashmessages() ?>
+<?php if (!empty($this->pdasubscription)) { ?>
+    <?=$this->render($this->infoText);?>
+    <h4><?=$this->bookDescription;?></h4>
+    <textarea style="resize: none" name="addressfield" autofocus="true"  form="edit-save-form" rows="5" cols="30" 
+              required="true" placeholder="<?=$this->transEsc('Please enter your address here');?>"></textarea>
+    <p/>
+    <form id="edit-save-form" class="form-horizontal" method="post"
+          action="<?= $this->recordLink()->getActionUrl($this->driver, 'PDASubscribe') ?>" name="pdasubscribeRecord">
+        <input type="hidden" name="action" value="pdasubscribe"/>
+        <input class="btn btn-primary" type="submit" value="<?= $this->transEsc('PDASubscribe') ?>"/>
+    </form>
+<?php } else { ?>
+    <p><?=$this->transEsc("Subscription exists for this document");?>.</p>
+    <form id="edit-save-form" class="form-horizontal" method="post"
+          action="<?= $this->recordLink()->getActionUrl($this->driver, 'PDASubscribe') ?>" name="pdaunsubscribeRecord">
+        <input type="hidden" name="action" value="pdaunsubscribe"/>
```

Comment:
Here and many other places: Please use underscores to separate names.  E.g. pdaunsubscribe -> pda_unsubscribe  It's far easier to parse for human readers.